urlcache.go

```diff
@@ -80,20 +80,23 @@ func (c *urlcache) keepCurrent(currentDate time.Time) {
 		headResp, err := http.Head(c.url)
 		if err != nil {
 			log.Errorf("Unable to request modified of %v: %v", c.url, err)
+			goto sleep
```

Comment:
We can extract a method or use a timer to avoid `goto`, but the current code seems more straightforward to me.